kms/yubikey/yubikey.go

```diff
@@ -338,7 +359,11 @@ func (k *YubiKey) CreateAttestation(req *apiv1.CreateAttestationRequest) (*apiv1
 
 // Close releases the connection to the YubiKey.
 func (k *YubiKey) Close() error {
-	return errors.Wrap(k.yk.Close(), "error closing yubikey")
+	err := k.yk.Close()
+	if err == nil {
+		pivMap.Delete(k.card)
+	}
+	return errors.Wrap(err, "error closing yubikey")
```

Comment:
This surprised me a bit, but apparently `github.com/pkg/errors.Wrap` handles nil errs. Maybe slightly easier to understand if explicitly checking for it to be not nil?

Reply:
Fixed with 4817292